source/agora/test/Common.d

```diff
@@ -0,0 +1,101 @@
+/*******************************************************************************
+
+    Common classes and utilities shared between the network tests.
+    Classes and utilities in this module are considered higher level than
+    in Base.d.
+
+    Copyright:
+        Copyright (c) 2020 BOS Platform Foundation Korea
+        All rights reserved.
+
+    License:
+        MIT License. See LICENSE for details.
+
+*******************************************************************************/
+
+module agora.test.Common;
```

Comment:
'
it's not used anywhere and stands out, I'd remove it.
If you need to debug you can use the scope (failure) assert(0); hack.
'

it is used in BlockRewardCalc.getEachValidatorReward

Reply:
> That's what `Base` is for.

my plan was to put the higher level classes Validator/Enrollment manager to a new file as the module description suggests:

```
Common classes and utilities shared between the network tests.
    Classes and utilities in this module are considered higher level than
    in Base.d.
```

Base.d is already a monster with 2000+ lines of code. I can move those classes to Base.d though

Reply:
1. It would be hard to find that class if we just put it randomly into some of the test files from where we use it.
2. Someone already moved those classes into Base.d anyways in recent v0.x.x branch :man-facepalming:

Reply:
In that case can you rebase please.